guides/release-process.md

```diff
@@ -65,9 +65,13 @@ of Cypress. You can see the progress of the test projects by opening the status
 
 In the following instructions, "X.Y.Z" is used to denote the [next version of Cypress being published](./next-version.md).
 
+_Note: It is advisable to notify the team that the `develop` branch is locked down prior to beginning the release process_
+
 1. Install and test the pre-release version to make sure everything is working.
-    - Get the pre-release version that matches your system from the latest develop commit.
-    - Install the new version: `npm install -g <cypress.tgz path>`
+    - Access the `cdn.cypress.io` S3 bucket and download the pre-release TGZ from the `/beta` directory. Ensure you pull the version associated with the latest commit SHA from `develop` and matching your system's architecture.
```

Comment:
This should be a bot comment on the last commit message. No one should have to got to the CDN